<launch>

    <group ns="/leviathan/description">


        <node pkg="stereo_image_proc" type="stereo_image_proc" name="stereo_image_proc" output="screen">
            <param name="approximate_sync" value="True" />
            <remap from="left/image_raw" to="camera_8/image_raw_adj"/>
            <remap from="right/image_raw" to="camera_9/image_raw_adj"/>
            <remap from="left/camera_info" to="camera_8/camera_info_adj"/>
            <remap from="right/camera_info" to="camera_9/camera_info_adj"/>
        </node>

        <node pkg="gazebo_drivers" type="cam_adjust.py" name="cam_adj_node"/>

    </group>


</launch>